token/jwt/claims_id_token.go

```diff
@@ -49,16 +49,39 @@ type IDTokenClaims struct {
 // ToMap will transform the headers to a map structure
```

Comment:
I have similar concerns about this as [here](https://github.com/ory/fosite/pull/520#discussion_r513301990).